frontend/src/pages/team/Settings/General.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <form class="space-y-6">
+    <div class="space-y-6">
```

Comment:
For this (and the some of other settings changes) semantically `form` does feel like a better fix than div. This part of the page does indeed contain a form, even if we're not using HTML form handling.

Reply:
very fair point - in most cases where we are using form - it isn't a form. Happy to revisit each of them and sanity check their usage.

Reply:
Whilst I do agree (which is why I used them originally), this is also a pragmatic fix for the issue as we don't rely on any of the browser-defined `<form>` behaviours.

We should aspire to restore `<form>` where appropriate - but will need to solve the UX issue around unexpected/inappropriate form submission.

Reply:
I'm :+1: to merge this as is and follow up (or create an issue) to restore the forms where appropriate.